package exceptions

object NoRAFound : Throwable() {

}
